syncmaster/worker/handlers/file/base.py

```diff
@@ -83,3 +78,42 @@ def _make_filter_expression(self, filters: list[dict]) -> str:
             expressions.append(f"{field} {op} '{value}'" if value is not None else f"{field} {op}")
 
         return " AND ".join(expressions)
+
+    def _make_columns_filter_expressions(self, filters: list[dict]) -> list[str] | None:
+        expressions = []
+        for filter in filters:
+            filter_type = filter["type"]
+            field = filter["field"]
+
+            if filter_type == "include":
+                expressions.append(field)
+            elif filter_type == "rename":
+                new_name = filter["to"]
+                expressions.append(f"{field} AS {new_name}")
```

Comment:
For Spark DataFrame (Hive and file sources), you can instead use:
```python
from pyspark.sql.functions import col

df.select(col("field"), col("field").cast("double").alias("field"), col("field").alias("new_name")
```

This automatically handles all column name quoting issues.

Just a recommendation, can be marked as TODO.